src/KeyHandler.cpp

```diff
@@ -48,6 +48,7 @@ constexpr double kNoOverrideThreshold = -8.0;
 
 static const char* GetKeyboardLayoutName(
     const Formosa::Mandarin::BopomofoKeyboardLayout* layout) {
+  // NOLINTBEGIN(readability-else-after-return)
```

Comment:
Hmm this one is a bit surprising. What if you move the last `return "Standard";` to a final else branch? Would both gcc and clang be happy with it?

Reply:
@lukhnos, I didn't find the rule is applicable here. It would prefer refactoring the following

```cpp
if () {
  return a;
} else if () {
  return b;
} else if () {
  return c;
}
...
```

into

```cpp
if () {
  return a;
}
if () {
  return b;
}
if () {
  return c;
}
...
```

Theoretically, they are identical, but I find this is a bit too lengthy and verbose. We are clearly returning different things w/ given layout.

I believe readability is all about letting the code reader spending minimum mental effort to understand the code intention, which we were clear. My decision is to suppress this rule locally, how do you think?

https://clang.llvm.org/extra/clang-tidy/checks/readability/else-after-return.html

Reply:
> I believe readability is all about letting the code reader spending minimum mental effort to understand the code intention, which we were clear. My decision is to suppress this rule locally, how do you think?

SGTM. I agree with your assessment about this particular rule.